k8s/sonarqube/schema.yaml

```diff
@@ -0,0 +1,57 @@
+application_api_version: v1beta1
+properties:
+  name:
+    type: string
+    x-google-marketplace:
+      type: NAME
+  namespace:
+    type: string
+    x-google-marketplace:
+      type: NAMESPACE
+  sonarqube.repository:
+    type: string
+    default: $REGISTRY:$TAG
+    x-google-marketplace:
+      type: IMAGE
+  sonarqube.volumeSize:
+    type: integer
+    title: Volume
+    description: The size of Volume SonarQube Data (Gi) in the solution
```

Comment:
Do we need to set it? Why user would need it?

Reply:
I thought it was good idea